contracts/cw-proposal-single/src/contract.rs

```diff
@@ -33,7 +34,7 @@ use crate::{
     state::{Ballot, BALLOTS, CONFIG, PROPOSALS, PROPOSAL_COUNT, PROPOSAL_HOOKS, VOTE_HOOKS},
 };
 
-pub(crate) const CONTRACT_NAME: &str = "crates.io:cw-govmod-single";
+pub(crate) const CONTRACT_NAME: &str = "crates.io:cw-proposal-single";
```

Comment:
**question**: are we removing usage of `govmod`? there is still usage of `govmod` in `packages/cw-core-macros`

Reply:
great catch! fixed.